packages/sdk/src/sway/predicates/predicateFinder.ts

```diff
@@ -0,0 +1,41 @@
+import { decompressBytecode } from 'fuels';
+
+export function getLatestPredicateVersion() {
+  // load versions
+  const versions = require('./versions.json');
```

Comment:
Prefer using ESModules instead of the CommonJS standard.

```suggestion
import { decompressBytecode } from 'fuels';
import versions from './versions.json';

export function getLatestPredicateVersion() {
```

or

```suggestion
import { decompressBytecode } from 'fuels';

export function getLatestPredicateVersion() {
  // load versions
  const versions = await import('./versions.json');
```

Reply:
good suggestion

the 1st is better, because we cant use async functions here, to prevent an factory by instance predicate

we need keep instance on `new Vault()`